src/nanoemoji/maximum_color.py

```diff
@@ -0,0 +1,179 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Generates color tables to provide support for a wide range of browsers.
+
+Requires input font have vector color capability, that is either an SVG table
+or COLR/CPAL.
+
+https://www.youtube.com/watch?v=HLddvNiXym4
```

Comment:
LOL

I was about to object to the name "maximum_color" then I watched the YT video and changed my mind 🤣 
Let's keep it!

Reply:
rofl victory